templates/default.mustache

```diff
@@ -4,6 +4,8 @@
 " by Chris Kempson (http://chriskempson.com)
 " {{scheme-name}} scheme by {{scheme-author}}
 
+" This file was generated from templates/default.mustache
```

Comment:
Where does this come from?

Reply:
Ah, ok you added it. I think this is not useful, because this file was not generated from a template. The readme are a better place for comments regarding how the project is built, here this is a bit ambiguous.

Reply:
Oh right, didn't think of that… I'll drop the commit and add something in the ~readme~ contributing.md